source/System.cpp

```diff
@@ -285,7 +288,14 @@ void System::Load(const DataNode &node, Set<Planet> &planets)
 				fleets.emplace_back(fleet, child.Value(valueIndex + 1), child);
 		}
 		else if(key == "raid")
-			RaidFleet::Load(raidFleets, child, remove, valueIndex);
+		{
+			child.PrintTrace("Warning: Deprecated use of \"raid\" instead of providing \"raiders\":");
+			if(!raiders)
+				raiders = new Raiders();
+			const_cast<Raiders *>(raiders)->LoadFleets(child, remove, valueIndex);
+		}
+		else if(key == "raiders")
+			raiders = GameData::GetRaiders().Get(child.Token(valueIndex));
```

Comment:
```suggestion
			raiders = GameData::GetRaiders().Get(value);
```

Reply:
we're not setting value up earlier and I don't really wanna change that here

Reply:
~~you literally have it used two branches before~~

Reply:
oh I was in the wrong file my bad